packages/rest/lib/http-decorators.cadl

```diff
@@ -2,45 +2,106 @@ namespace Cadl.Http;
 
 using Cadl.Reflection;
 
+/**
+ * Header options.
+ */
+model HeaderOptions {
+  /**
+   * Name of the header when sent over http.
+   */
+  name?: string;
+  /**
+   * How the parameter value will be serialized
+   */
+  format?: "simple";
```

Comment:
I thought from the design we wanted to have this be `csv` not simple?

Reply:
We could, but in that case, I think the decorator option will not be able to describe all possible formats in OpenAPI3.

For example, with `pipeDelimited` format: `blue|black|brown` for array, `R|100|G|200` for object.

Not sure if this has been considered?

Reply:
Confirmed with @markcowl , for now we'll only consider supporting limited formats in Cadl. I'll update the implementation accordingly.